OpenOversight/app/utils/forms.py

```diff
@@ -86,46 +86,46 @@ def add_officer_profile(form: AddOfficerForm, user: User) -> Officer:
     )
     db.session.add(assignment)
     if form.links.data:
-        for link in form.data["links"]:
-            if link["url"]:
-                li = get_or_create_link_from_form(link, user)
-                officer.links.append(li)
+        form_links = [link for link in form.data["links"] if link["url"]]
+        for link in form_links:
```

Comment:
Not sure I would agree with the changes in this file, but also don't feel strongly about it. Was this something ruff suggested?

Reply:
I made a handful of changes due to [this rule](https://www.flake8rules.com/rules/C901.html), and this file was one of them. It called out 10+ other functions that had more than 10 if/else statements, and I changed the ones that could be done with the least amount of re-working. 

If you'd like to see the other functions, you could remove `C901` from the list and see what I mean.

Reply:
I think we could reimplement `C901` in another PR, but it would likely require a lot of changes to functions within our codebase:
```console
ruff.....................................................................Failed
- hook id: ruff
- exit code: 1

OpenOversight/app/commands.py:183:5: C901 `update_officer_from_row` is too complex (14 > 10)
OpenOversight/app/commands.py:473:5: C901 `bulk_add_officers` is too complex (14 > 10)
OpenOversight/app/csv_imports.py:156:5: C901 `_handle_assignments_csv` is too complex (18 > 10)
OpenOversight/app/main/model_view.py:113:9: C901 `edit` is too complex (11 > 10)
OpenOversight/app/main/views.py:745:5: C901 `edit_department` is too complex (15 > 10)
OpenOversight/app/main/views.py:883:5: C901 `list_officer` is too complex (18 > 10)
OpenOversight/app/main/views.py:1374:5: C901 `label_data` is too complex (11 > 10)
OpenOversight/app/models/database_imports.py:102:5: C901 `update_officer_from_dict` is too complex (11 > 10)
OpenOversight/app/utils/forms.py:273:5: C901 `filter_by_form` is too complex (21 > 10)
Found 9 errors.

black....................................................................Passed
mypy.....................................................................Passed
```